src/modules/shell.rs

```diff
@@ -28,6 +28,16 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                 },
                 _ => None,
             })
+            .map(|var| match var {
+                "bash_indicator" => Some(Ok(config.bash_indicator)),
+                "fish_indicator" => Some(Ok(config.fish_indicator)),
+                "zsh_indicator" => Some(Ok(config.zsh_indicator)),
+                "powershell_indicator" => Some(Ok(config.powershell_indicator)),
+                "ion_indicator" => Some(Ok(config.ion_indicator)),
+                "elvish_indicator" => Some(Ok(config.elvish_indicator)),
+                "tcsh_indicator" => Some(Ok(config.tcsh_indicator)),
+                _ => None,
+            })
```

Comment:
It seems like there should be a better way to do this, but I'm in a little over my head. Very open to better ideas!

Reply:
Hmm, yeah it is repetitive and verbose but it might just be **the** way according to https://users.rust-lang.org/t/access-struct-attributes-by-string/17520/2